Nasal/c172p.nas

```diff
@@ -623,3 +624,16 @@ setlistener("/sim/signals/fdm-initialized", func {
     var c172_timer = maketimer(0.25, func{global_system_loop()});
     c172_timer.start();
 });
+
+var ControlLock = func {
+
+    if (getprop("/sim/model/c172p/cockpit/control-lock-placed")) {
+        setprop("/sim/model/c172p/cockpit/yoke-aileron", 0);
+        setprop("/sim/model/c172p/cockpit/yoke-elevator", 0);
+        setprop("/controls/flight/aileron", 0);
+        setprop("/controls/flight/elevator", 0);
+    }
+
+    settimer(ControlLock,0);
```

Comment:
I don't like this `settimer()` call here. At the very least you should be using `maketimer()` for periodic tasks. But I would suggest to use a `<filter>` in an `<autopilot>` file since that is executed at FDM rate. Currently I can still move the control surfaces a bit until Nasal wakes up, drinks some coffee, and then resets the surfaces to 0.